docs/samples/Microsoft.ML.Samples/Dynamic/MinMaxNormalizer.cs

```diff
@@ -35,7 +35,16 @@ public static void MinMaxNormalizer()
             // A pipeline for normalizing the Induced column.
             var pipeline = ml.Transforms.Normalize("Induced");
             // The transformed (normalized according to Normalizer.NormalizerMode.MinMax) data.
-            var transformedData = pipeline.Fit(trainData).Transform(trainData);
+            var transformer = pipeline.Fit(trainData);
+
+            var modelParams = transformer.ModelParameters["Induced"] as NormalizerTransformer.IAffineData<float>;
+           Console.WriteLine($"The normalization parameters are: Scale = {modelParams.Scale} and Offset = {modelParams.Offset}");
+            //Preview 
+            //
+            //The normalization parameters are: Scale = 0.5 and Offset = 0"
```

Comment:
is this some standard notation for documentation to pick up? Or just a comment?

Reply:
Typically they put on comments the output of a printout, or result. 

See the example in this: https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.buildengine.buildproperty?view=netframework-4.7.2

Reply:
Typically they put on comments the output of a printout, or result. 

See the example in this: [https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.buildengine.buildproperty?view=netframework-4.7.2](https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.buildengine.buildproperty?view=netframework-4.7.2) 

---
In reply to: [233685871](https://github.com/dotnet/machinelearning/pull/1594#discussion_r233685871) [](ancestors = 233685871)